Add ROS1_MIX_GENERATOR specific option for ROS 1 (#21)
eProsima/ROS1-SH: README.md

  ~/is_ws$ colcon build --cmake-args -DBUILD_ROS1_TESTS=ON
  ```

* `MIX_ROS_PACKAGES`: It accepts as an argument a list of [ROS packages](https://index.ros.org/packages/),
  such as `std_msgs`, `geometry_msgs`, `sensor_msgs`, `nav_msgs`... for which the required transformation

>   library to convert the specific ROS 1 type definitions into *xTypes*, and the other way around, will be built.

Reviewer: ```suggestion
  library to convert the specific ROS 1 type definitions into *xTypes*, and the other way around will be built.
```

Author: makes no sense to remove it

  This list is shared with the [ROS 2 System Handle](https://github.com/eProsima/ROS2-SH#compilation-flags),
  meaning that the ROS packages specified in the `MIX_ROS_PACKAGES` variable will also be built for *ROS 2*

>   if this *System Handle* is present within the *Integration Service* workspace. If a certain package
>   is only present in *ROS 1*, the `MIX_ROS1_PACKAGES` flag must be used instead.

Reviewer: ```suggestion
  if the corresponding *System Handle* is present within the *Integration Service* workspace. To avoid errors, if a certain package
  is only present in *ROS 1*, the `MIX_ROS1_PACKAGES` flag must be used instead.
```

  These transformation libraries are also known within the *Integration Service* context as `Middleware Interface Extension`
  or `mix` libraries.

  By default, only the `std_msgs_mix` library is compiled, unless the `BUILD_TESTS`
  or `BUILD_ROS1_TESTS` is used, case in which some additional ROS 1 packages `mix` files
  required for testing will be built.

  If the user wants to compile some additional packages to use them with *Integration Service*,
  the following command must be launched to compile it, adding as much packages to the list as desired:
  ```bash
  ~/is_ws$ colcon build --cmake-args -DMIX_ROS_PACKAGES="std_msgs geometry_msgs sensor_msgs nav_msgs"
  ```

* `MIX_ROS1_PACKAGES`: It is used just as the `MIX_ROS_PACKAGES` flag, but will only affect *ROS 1*;

>   this means that the `mix` generation engine will not look for *ROS 2* packages present in this list,

Reviewer: ```suggestion
  this means that the `mix` generation engine will not search within the *ROS 2* packages,
```
